src/js/components/agencyV2/visualizations/totalObligationsOverTime/axis/AxisLabel.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * AxisLabel.jsx
```

Comment:
related to the file structure convo we havent really had yet and your changes to the `src/api/` dir...

I dont like having files like this. This is a tiny component that could just be defined inside the relevant parent. If it's reused maybe we could export it but otherwise it doesnt make sense to have a new directory in the source code for components that are this small.

Reply:
Yes, I have seen your PR's that include multiple components or even multiple component with multiple functions. Including multiple components in one file is confusing and unless it is a file like `helperFunctionForVis` or `dataMappingForCOVID` it is not best practice.  This component is used by both the x and y axis.